### PR TITLE
Small ui improvements

### PR DESCRIPTION
**Pull request by jonasdt**
For details see commit messages.

Remark regarding bbe394e:  There was a bug in the container css that caused the left panel to disappear sometimes, depending on the content and the size of the window. Sometimes it happened after closing the chrome developers tools.

### CODE REVIEW

File: src/Whoops/Resources/js/whoops.base.js [hunk 1]
```diff
@@ -94,20 +104,34 @@ Zepto(function($) {
     return actionMsg;
   }
 
+  function scrollIntoView($node, $parent) {
+    var nodeOffset = $node.offset(),
+        nodeTop = nodeOffset.top,
```
Comment (denis-sokolov) [inline src/Whoops/Resources/js/whoops.base.js]: Regardless of personal preference, the surrounding code uses one `var` per variable style, so new code should do that, too.

- [ ] Use consistent var style

File: src/Whoops/Resources/js/whoops.base.js [hunk 2]
```diff
@@ -94,20 +104,34 @@ Zepto(function($) {
     return actionMsg;
   }
 
+  function scrollIntoView($node, $parent) {
```
Comment (denis-sokolov) [inline src/Whoops/Resources/js/whoops.base.js]: Although the commit message is supposed to explain something, I still don't fully understand what this does differently. Could you explain?

- [ ] Clarify custom `scrollIntoView`

**Comment by denis-sokolov** [comment]
Great, including `j/k`. Needs extra testing due to the amount of layout changes, though.

File: src/Whoops/Resources/js/whoops.base.js (hunk 2 above)
Comment (jonasdt) [inline src/Whoops/Resources/js/whoops.base.js]: The old code always keeps the active frame at the top. Check the blue line in the active frame, the first time it makes a jump, afterwards it stays on top
https://www.dropbox.com/s/s916mej3hva6fk6/before.mov?dl=0

The new code does what you would expect to do
https://www.dropbox.com/s/tjh95qj52xkwd3u/after.mov?dl=0

File: src/Whoops/Resources/js/whoops.base.js (hunk 1 above)
Comment (jonasdt) [inline src/Whoops/Resources/js/whoops.base.js]: Done

**Comment by denis-sokolov** [comment]
Thank you for your contribution!

**Pull request merged**